cmd/project.go

```diff
@@ -67,9 +67,61 @@ func newProjectCommand(conf *config.Config, ctx *cli.Context, workDir string) *P
 	}
 }
 
+func (p *ProjectCommands) createProjectFile() error {
+	var buf bytes.Buffer
+
+	if !p.Ctx.IsSet("environments") || !p.Ctx.IsSet("scopes") {
+		return fmt.Errorf("%s file not found or values not set for flags: %s, %s",
+			util.GetPwdPath(util.TenantProjectFile), "environments", "scopes")
+	}
+
+	if p.Ctx.IsSet("environments") {
+		p.projectFile.Spec.Environments = make(map[string]*config.ProjectRootDomain)
+		for _, val := range p.Ctx.StringSlice("environments") {
+			if len(val) > 0 {
+				matchRootDomain := regexp.MustCompile(`^.+\.root-domain=.+$`).MatchString(val)
+				splitRootDomain := strings.SplitN(val, ".", 2)
+
+				if !matchRootDomain && len(splitRootDomain) == 2 {
+					return fmt.Errorf("option %s for environment %s set not correctly",
```

Comment:
`"option %s for environment %s not set correctly"`

Reply:
fixed